pylint/pyreverse/main.py

```diff
@@ -139,7 +152,10 @@
             action="store",
             default="dot",
             metavar="<format>",
-            help="create a *.<format> output file if format available.",
+            help=(
+                f"create a *.<format> output file if format is available. Available formats are: {', '.join(DIRECTLY_SUPPORTED_FORMATS)}. "
+                f"Any other format will be tried to create by means of the 'dot' command line tool, which requires a graphviz installation."
+            ),
```

Comment:
We did not state the directly supported output formats in the help output previously, I think this also makes it a bit clearer what is going on under the hood.